### PR TITLE
Add the ability to generate SSL certs from tray icon

### LINKED ISSUE #735: Rework the HTTPS certs install: add menu options in the tray menu
The certificates should be handled only by the agent and not by the installer. This has to be done using a menu in the tray icon “generate and install HTTPS certs” and “remove HTTPS certs”. The certs do not have to be written in the same directory of the executable, but in a temporary folder.

### PR DESCRIPTION
**Pull request by umbynos**
**Please check if the PR fulfills these requirements**

- [X] The PR has no duplicates (please search among the [Pull Requests](https://github.com/arduino/arduino-create-agent/pulls)
      before creating one)
- [ ] Tests for the changes have been added (for bug fixes / features)

* **What kind of change does this PR introduce?**

feature

- **What is the current behavior?**

Fix #735 
* **What is the new behavior?**

There is a new clickable voice in the tray icon's menu that allows a Mac/~~windows~~ user to generate and install in the keychain the SSL certificates that are used for the agent to work in HTTPS mode.
Linux has this menuItem disabled, since the certificates are not required on this OS.
image
The voice is clickable only if the certs have not already been generated, otherwise it's grayed-out
- **Does this PR introduce a breaking change?**

yes, the installation of the certs have been removed from the installer. It is now available from the tray icon. The doc have to be updated, if the user wants to remove the installed cert it has to be done manually using the keychain management tool of the system.
* **Other information**:

In the installer, we should remove the cert handling. (the 3 different flavors of the agent are no more required), also the CI needs some changes

### CODE REVIEW

**Comment by umbynos** [comment]
missing windows implementation of `install_windows.go`

**Comment by umbynos** [comment]
TODO: the menu icon shouldn't be visible on linux, since the certs install is not required on this platform

**Comment by umbynos** [comment]
Test release [here](https://github.com/arduino/arduino-create-agent/releases/tag/1.2.8-rc1)

File: certificates/install_windows.go [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+	// Decode PEM
+	if certBlock, _ := pem.Decode(cert); certBlock == nil || certBlock.Type != "CERTIFICATE" {
+		log.Error("invalid PEM data: decode pem")
+	} else {
+		cert = certBlock.Bytes
+	}
+	// Open root store
+	store, err := openWindowsRootStore()
+	if err != nil {
+		log.Errorf("cannot open root store %s", err)
```
Comment (umbynos) [inline certificates/install_windows.go:54]: Maybe it would be better to open a dialog with an error, or something like that

Comment (umbynos) [reply certificates/install_windows.go:54]: same for macos

**Comment by umbynos** [comment]
I've removed support for windows in 9f7c05a since the newer version of edge is chromium based: chromium and chrome do not require the agent in HTTPS

File: certificates/install_windows.go (hunk 1 above)
Comment (umbynos) [reply certificates/install_windows.go]: Done in [a78e9a5](https://github.com/arduino/arduino-create-agent/pull/778/commits/a78e9a515ed6d8704962700e969f7797dd9546c5)

**Review by cmaglie — APPROVED** [review]

**Pull request merged**